samples/servicecontrol/adapter-sqlserver/sample.md

```diff
@@ -0,0 +1,108 @@
+
+This topology is used in business endpoints (Sales, Shipping) as well as in the Adapter.
+
+### Sales and Shipping
+
+The Sales and Shipping projects contain endpoints that simulate execution of business process. The process consists of two events: `OrderAccepted` published by Sales and subscribed by Shipping and `OrderShipped` published by Shipping and subscribed by Sales.
+
+The Sales and Shipping endpoints use separate databases and their transports are configured in the [multi-instance](/nservicebus/sqlserver/deployment-options.md#modes-overview-multi-instance) mode using the topology definition from the `Connections` class.
+
+The business endpoints include a database failure simulation mode (toggled by pressing `f`) which can be used to generate failed messages for demonstrating message retry functionality.
+
+The Shipping endpoint has the Heartbeats plug-in installed to enable uptime monitoring via ServicePulse.
+
+### Adapter
+
+The Adapter project hosts the `ServiceControl.TransportAdapter`. The adapter has two sides: endpoint-facing and ServiceControl-facing. Each side can use different transport but in this sample both use SQL Server transport:
+
+snippet: AdapterTransport
+
+The endpoint-facing side config enables the [multi-instance](/nservicebus/sqlserver/deployment-options.md#modes-overview-multi-instance) mode of SQL Server transport using the shared topology.
+
+snippet: EndpointSideConfig
+
+The ServiceSontrol-facing side config specifies the connection string to the database used by ServiceControl:
+
+snippet: SCSideConfig
+
+Because the ServiceControl has been installed under a non-default instance name (`Particular.ServiceControl.SQL`) the control queue name needs to be overridden in the adapter configuration:
+
+snippet: ControlQueueOverride
+
+## How it works
+
+The purpose of the adapter is to isolate ServiceControl from the specifics of physical deployment topology of the business endpoints (such as [multi-instance](/nservicebus/sqlserver/deployment-options.md#modes-overview-multi-instance) mode. In order to do so, the adapter presents to the business endpoints the same interface as regular service control (in form of `audit`, `error` and `Particular.ServiceControl`) queues.
+
+### Heartbeats
+
+The heartbeat messages arrive at adapter's `Particular.ServiceControl` queue. From there they are moved to the `Particular.ServiceControl.SQL` queue in the ServiceControl database. In case of problems (e.g. the destination database is down) the forward attempts are repeated a configurable number of times after which messages are dropped to prevent the queue from growing indefinitely.
+
+### Audits
+
+The audit messages arrive at adapter's `audit` queue. From there there are moved to `audit` queue in ServiceControl database and ingested by ServiceControl.
+
+### Retries
```

Comment:
The poison queue is not used in the business message retries path, only for audits. I'd rather cover it in the doco.